src/utils/phoenix-utils.js

```diff
@@ -1,15 +1,14 @@
-import queryString from "query-string";
 import uuid from "uuid/v4";
 import { Socket } from "phoenix";
 
 export function connectToReticulum() {
-  const qs = queryString.parse(location.search);
+  const qs = new URLSearchParams(location.search);
 
-  const socketProtocol = qs.phx_protocol || (document.location.protocol === "https:" ? "wss:" : "ws:");
+  const socketProtocol = qs.get("phx_protocol") || (document.location.protocol === "https:" ? "wss:" : "ws:");
   const [retHost, retPort] = (process.env.DEV_RETICULUM_SERVER || "").split(":");
   const isProd = process.env.NODE_ENV === "production";
-  const socketPort = qs.phx_port || (isProd ? document.location.port : retPort) || "443";
-  const socketHost = qs.phx_host || (isProd ? document.location.hostname : retHost) || "";
+  const socketPort = qs.get("phx_port") || (isProd ? document.location.port : retPort) || "443";
+  const socketHost = qs.get("phx_host") || (isProd ? document.location.hostname : retHost) || "";
   const socketUrl = `${socketProtocol}//${socketHost}${socketPort ? `:${socketPort}` : ""}/socket`;
```

Comment:
would it make sense to allow passing in a `?phx_url`

Reply:
That does seem more straightforward. I see no good reason not to do it. I will consider performing that transformation after I dig up where anyone might be using these parameters.